backend/main.py

```diff
@@ -2,17 +2,26 @@
 from fastapi.staticfiles import StaticFiles
 from app.api.v1.api import router as api_router
 import uvicorn
+import os
 
 app = FastAPI(
     title="Hermes API",
     version="1.0.0",
     docs_url="/docs",
 )
 
-app.mount("/", StaticFiles(directory="../frontend/dist", html=True), name="static")
+# Use a relative path for the Docker container
+static_dir = "frontend/dist"
+if not os.path.exists(static_dir):
+    raise RuntimeError(f"Directory '{static_dir}' does not exist")
+
+app.mount("/static", StaticFiles(directory=static_dir, html=True), name="static")
```

Comment:
What files are we looking for?

Reply:
Honestly uvicorn was not executing without this block, maybe in a future ticket we can look more at this. What do you think?